setGameMac/GameLogic/SetGame.swift

```diff
@@ -2,22 +2,203 @@
 //  SetGame.swift
 //  setGameMac
 //
-//  Created by Kristina on 26.10.2021.
-//
+//  Created
 
-import Foundation
 
 struct SetGame {
     
     private(set) var cardsInDeck = [Card]()
     private(set) var cardsInGame = [Card]()
+    private(set) var discardPile = [Card]()
+    
+    private var chosenCards : [Card]? {
+        get {
+            let chosenCardIndices = cardsInGame.indices.filter {cardsInGame[$0].isSelected}
+            return chosenCardIndices.count > 0 ? chosenCardIndices.map { cardsInGame[$0] } : nil
+        }
```

Comment:
you can avoid `get` in such code style.
it will work even if you remove explicit getter declaration.